CedarJava/src/main/java/com/cedarpolicy/value/EntityTypeName.java

```diff
@@ -34,7 +32,15 @@ public final class EntityTypeName {
     protected EntityTypeName(List<String> namespace, String  basename) {
         this.namespace = namespace;
         this.basename= basename;
-        this.entityTypeNameRepr = Suppliers.memoize(() -> getEntityTypeNameRepr(this));
+        this.entityTypeNameRepr = new Supplier<String>() {
```

Comment:
If anyone is wondering why this is stalled: The Guava memoizing supplier is thread safe, but this implementation would not be. We don't make an explicit guarantees about thread safety of the Java bindings, but out Rust implementation _is_ thread safe, so I don't want to break that here, and I haven't had the time to do this properly.